keps/sig-scheduling/902-non-preempting-priorityclass/README.md

```diff
@@ -164,15 +165,18 @@ Ensure existing tests (for preempting PriorityClasses) do not break.
 ### Graduation Criteria
 #### Alpha (v1.15):
 
-- [x] Support NonPreemptingPriority in PriorityClasses
+- Support NonPreemptingPriority in PriorityClasses
 
 #### Beta (v1.19):
 
-- [ ] Add integration test for NonPreemptingPriority.
-- [ ] Graduate NonPreemptingPriority to Beta.
-- [ ] Update documents to reflect the changes.
-
+- Add integration test for NonPreemptingPriority.
+- Graduate NonPreemptingPriority to Beta.
+- Update documents to reflect the changes.
 
+#### Stable (v1.24):
+- No negative feedback.
+- Graduate NonPreemptingPriority to GA.
+- Update documents to reflect the changes.
```

Comment:
I can't comment on the lines below so commenting here:

L196: "Yes, the feature can be disabled if the PreemptionPolicy isn't set.". 

What if PreemptionPolicy is set? 
[Technically, with GA we will lock it to true, but it requires clarifying.]
I would assume (this is what we generally do for features), that the field remains set, but is ignored by components.


L206: nit: I'm assuming you mean "crashing", right?

L210: it's not N/A - let's mention scheduler crashes and errors (there is a metric where you report errors IIRC)
FWIW - you're also mentioning some metrics in troubleshooting section, which sound like a good fit here too.

L213: it's definitely not N/A
  It's a bit too late, but still, can you please do some manual tests for it?

Reply:
Thanks. I add more information about Production Readiness.

Reply:
Should we include the new questions from the template too?

For example:

- How can an operator determine if the feature is in use by workloads?
- How can someone using this feature know that it is working for their instance?

Reply:
Done

Reply:
Thanks!